docs/Installation.md

```diff
@@ -58,11 +58,7 @@ If you'd like to use Docker for ML-Agents, please follow
 
 ## Unity Packages
 
-You can download ML-Agents as Unity Packages:
-
- * [ML-Agents with TensorflowSharp Plugin](https://s3.amazonaws.com/unity-ml-agents/0.3/ML-AgentsWithPlugin.unitypackage)
- * [ML-Agents without TensorflowSharp Plugin](https://s3.amazonaws.com/unity-ml-agents/0.3/ML-AgentsNoPlugin.unitypackage)
- * [TensorflowSharp Plugin Only](https://s3.amazonaws.com/unity-ml-agents/0.3/TFSharpPlugin.unitypackage)
+You can download the TensorflowSharp Plugin as a Unity package [here](https://s3.amazonaws.com/unity-ml-agents/0.3/TFSharpPlugin.unitypackage).
```

Comment:
TensorflowSharp --> [TensorFlowSharp](Background-TensorFlow.md#tensorflowsharp) 
- Capitalize F
- Add link for the uninitiated

Plugin --> plugin